README.md

```diff
@@ -351,6 +363,160 @@ Omnichannel offers an live chat widget (LCW) by default. You can use the Chat SD
+        }
+    }
+    const chatSDK = new OmnichannelChatSDK.OmnichannelChatSDK(omnichannelConfig, chatSDKConfig);
+    await chatSDK.initialize();
+    
+    ....
+    
+    const chatReconnectContext = await chatSDK?.getChatReconnectContext(); 
+    
+    if (chatReconnectContext.reconnectId)  {
+        reconnectId = chatReconnectContext.reconnectId;
+    }
+    
+    ....
+    
+    const optionalParams: any = {};
+   
+    optionalParams.reconnectId = reconnectId
+```
+### Authenticated Chat Reconnect using URL
+ 
+```ts
+    const chatSDKConfig = {
+        chatReconnect: {
+            disable: false,
+        },
+        getAuthToken: async () => {
+            const response = await fetch("http://contosohelp.com/token");
+            if (response.ok) {
+                return await response.text();
+            }
+            else {
+                return null
+            }
+        }
+    }
+
+    const chatSDK = new OmnichannelChatSDK.OmnichannelChatSDK(omnichannelConfig, chatSDKConfig);
+    await chatSDK.initialize();
+    
+    ....
+    
+    const optionalParams: any = {};
+    
+    //retrieve the reconnect ID from the URL
+    const  retrieveReconnectId = function(): any {
+        const urlParams = new URLSearchParams(window.location.search);
+
+        if (urlParams.get('oc.reconnectid') !== null) {
+            return urlParams.get('oc.reconnectid');
+        } else {
+            return null;
+        }
+    }
+     
+    const params = {
+        reconnectId: retrieveReconnectId(),
+    }
+
+    const chatReconnectContext = await chatSDK?.getChatReconnectContext(params); 
+
+    if ( chatReconnectContext !== null && chatReconnectContext !== undefined) {
+        if (chatReconnectContext.redirectURL !== null && chatReconnectContext.redirectURL !== undefined) {
+            window.location.replace(chatReconnectContext.redirectURL);
+        } else {
+            optionalParams.reconnectId = chatReconnectContext.reconnectId
+        }
+    }
+    
+    await chatSDK?.startChat(optionalParams);
+```
+### Unauthenticated Chat Reconnect
+
+```ts
+    const chatSDKConfig = {
+        chatReconnect: {
+            disable: false,
+        },
+    }
+
+    const chatSDK = new OmnichannelChatSDK.OmnichannelChatSDK(omnichannelConfig, chatSDKConfig);
+    await chatSDK.initialize();
+
+    ....
+    
+    const optionalParams: any = {};
+    
+    //retrieve the reconnect ID from the URL
+    const  retrieveReconnectId = function(): any {
+        const urlParams = new URLSearchParams(window.location.search);
+
+        if (urlParams.get('oc.reconnectid') !== null) {
+            return urlParams.get('oc.reconnectid');
+        } else {
+            return null;
+        }
+    }
+     
+    const params = {
+        reconnectId: retrieveReconnectId(),
+    }
+
+    const chatReconnectContext = await chatSDK?.getChatReconnectContext(params); 
+
+    if ( chatReconnectContext !== null && chatReconnectContext !== undefined) {
```

Comment:
// Invalid reconnect id, redirect URL if any
    if (chatReconnectContext.redirectURL) {
        window.location.replace(chatReconnectContext.redirectURL);
    } 
    
    // Valid reconnect id, reconnect to previous chat
    if (chatReconnectContext.reconnectId) {
        await chatSDK.startChat({
            reconnectId: chatReconnectContext.reconnectId
        });
    } else {  // ReconnectId from URL is not valid, start new chat session
        await chatSDK.startChat();
    }